source/client-side-encryption/tests/README.rst

```diff
@@ -769,4 +769,174 @@ The following tests that setting ``bypassAutoEncryption=true`` really does bypas
 
 #. Use ``client_encrypted`` to insert the document ``{"unencrypted": "test"}`` into ``db.coll``. Expect this to succeed. 
 
-#. Validate that mongocryptd was not spawned. Create a MongoClient to localhost:27021 (or whatever was passed via ``--port``) with serverSelectionTimeoutMS=1000. Run an ``isMaster`` command and ensure it fails with a server selection timeout.
\ No newline at end of file
+#. Validate that mongocryptd was not spawned. Create a MongoClient to localhost:27021 (or whatever was passed via ``--port``) with serverSelectionTimeoutMS=1000. Run an ``isMaster`` command and ensure it fails with a server selection timeout.
+
+Deadlock tests
+~~~~~~~~~~~~~~
+
+There are multiple parameterized test cases. Before each test case, perform the setup.
```

Comment:
It may be worth adding a note to explain that these tests don't apply for drivers that don't implement connection pools (e.g. PHP)

Reply:
Done.